mkdocs.yml

```diff
@@ -0,0 +1,96 @@
+docs_dir: docs/
+repo_url: https://github.com/cloud-bulldozer/rosa-burner
+nav:
+- Home: index.md
+- Platforms:
+  - platforms/index.md
+  - Rosa:
+    - platforms/rosa/index.md
+    - Terraform: platforms/rosa/terraform.md
+    - Hypershift: platforms/rosa/hypershift.md
+- Measurements: measurements.md
+- Observability & alerting:
+  - observability/index.md
+  - Overview: observability/index.md
+  - Indexing: observability/indexing.md
```

Comment:
If we plan on adding content here, please update these to below:

```
- Observability & alerting:
  - docs/observability/index.md
  - Overview: docs/observability/index.md
  - Indexing: docs/observability/indexing.md
```

Reply:
removed